src/browser-lib/fixtures/footerSections.ts

```diff
@@ -3,36 +3,73 @@ import { FooterSections } from "../../components/SiteFooter/SiteFooter";
 const footerSections: FooterSections = {
   pupils: {
     title: "Pupils",
-    links: [{ text: "Classroom", type: "pupils-link" }],
+    links: [
+      {
+        text: "Learn online",
+        type: "page",
+        page: "classroom",
+        icon: "external",
+      },
+    ],
   },
   teachers: {
     title: "Teachers",
     links: [
-      { text: "Teacher Hub", type: "teachers-link" },
-      { text: "Plan a lesson", href: "/lesson-planning" },
-      { text: "Develop your curriculum", href: "/develop-your-curriculum" },
-      { text: "Support your team", href: "/support-your-team" },
+      // {  // Commented out until launch
+      //   text: "Key stage 1",
+      //   href: "/teachers/key-stages/ks1/subjects",
+      // },
+      // {
+      //   text: "Key stage 2",
+      //   href: "/teachers/key-stages/ks2/subjects",
+      // },
+      // {
+      //   text: "Key stage 3",
+      //   href: "/teachers/key-stages/ks3/subjects",
+      // },
+      // {
+      //   text: "Key stage 4",
+      //   href: "/teachers/key-stages/ks4/subjects",
+      // },
+
+      // {
+      //   text: "Curriculum plans",
+      //   type: "page",
+      //   page: "curriculum-landing-page",
+      // },
```

Comment:
Do you know if we're tracking all these bits of cleanup work we'll need for launch?

Reply:
Great idea, i've added one here
https://www.notion.so/oaknationalacademy/Dev-s-pre-launch-clean-up-9facc788b59542a6b9f2baacfa63ff30